src/packages/excalidraw/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ Please add the latest change on the top under the correct section.
 
 ## Unreleased
 
+## Types
+
+- Renamed [#3427](https://github.com/excalidraw/excalidraw/pull/3427)
+  - `DataState` → `ExportedDataState`
+  - `LibraryData` → `ExportedLibraryData`
+
```

Comment:
I think Excalidraw API should be the first header under changelog so can we move it after excal API?
And it should be under one of [semantic types](https://github.com/commitizen/conventional-commit-types/blob/master/index.json) so `refactor` should be fine. Also maybe we should mention a note that this would be a breaking change if you are using the above types so you will need to rename it just like we do for prop renaming